src/Symfony/Component/Yaml/Unescaper.php

```diff
@@ -109,6 +114,8 @@ public function unescapeCharacter($value)
                 return self::utf8chr(hexdec(substr($value, 2, 4)));
             case 'U':
                 return self::utf8chr(hexdec(substr($value, 2, 8)));
+            default:
+                throw new ParseException(sprintf('Found unknown escape character (%s in \\%s).', $value, $value));
```

Comment:
As mentioned in https://github.com/symfony/symfony/pull/16201/files#r41711538, this is not possible.